src/helpers/dotEnv.ts

```diff
@@ -0,0 +1,6 @@
+export default {
+  ENV: process.env.NODE_ENV || 'development',
+  PORT: process.env.PORT || 3000,
+  ADMIN_PASS: process.env.ADMIN || null,
+  HOME: process.env.HOME || null,
```

Comment:
from my professional experience, its better to also have helper functions.... like `isProduction`